lib/obfuscate_id.rb

```diff
@@ -4,8 +4,9 @@ def obfuscate_id(options = {})
 
     extend ClassMethods
     include InstanceMethods
-    cattr_accessor :obfuscate_id_spin
+    cattr_accessor :obfuscate_id_spin, :obfuscater # Obfuscater, aka the ScatterSwapper for this spin
     self.obfuscate_id_spin = (options[:spin] || obfuscate_id_default_spin)
+    self.obfuscater = ScatterSwap::Hasher.new(self.obfuscate_id_spin)
   end
 
   def self.hide(id, spin)
```

Comment:
I think this is still right. I kept the old `ScatterSwap` static methods that take these params. See scatter_swap.rb for the definition of those methods.